src/Sulu/Bundle/AdminBundle/Resources/js/components/Divider/tests/__snapshots__/Divider.test.js.snap

```diff
@@ -1,15 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Render a Divider with text 1`] = `
-<div
-  class="divider"
->
-  Test
-</div>
-`;
+exports[`Render a Divider with text 1`] = `undefined`;
 
-exports[`Render an empty Divider 1`] = `
-<div
-  class="divider"
-/>
-`;
+exports[`Render an empty Divider 1`] = `undefined`;
```

Comment:
Looks something strange is happening here that the divider is undefined know.

Reply:
I was using the enzyme `render()` function instead of the one by RTL. 🙈 This is fixed in the latest commit.